packages/frontend/src/components/ProjectConnection/DbtForms/BitBucketForm.tsx

```diff
@@ -30,6 +38,10 @@ const BitBucketForm: FC<{ disabled: boolean }> = ({ disabled }) => (
             documentationUrl="https://docs.lightdash.com/get-started/setup-lightdash/connect-project#repository-1"
             rules={{
                 required: 'Required field',
+                validate: {
+                    hasWhiteSpaces: hasWhiteSpaces('Repository'),
```

Comment:
I feel like the naming is a bit confusing here, it sounds like we are expecting for it to have white spaces?